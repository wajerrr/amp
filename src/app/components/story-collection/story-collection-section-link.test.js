import React from 'react';
import renderer from 'react-test-renderer';
import SectionLink from './story-collection-section-link';

describe('SectionLink', () => {
  it('should match snapshot without className', () => {
    const tree = renderer
      .create(<SectionLink href="/">link</SectionLink>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should match snapshot with className', () => {
    const tree = renderer
      .create(
        <SectionLink href="/" className="class">
          link
        </SectionLink>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
