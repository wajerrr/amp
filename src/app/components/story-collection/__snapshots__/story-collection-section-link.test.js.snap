// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionLink should match snapshot with className 1`] = `
<a
  className="css-1uvd0mj class"
  href="/"
>
  link
  <span
    className={undefined}
  >
    <svg
      version="1.1"
      viewBox="0 0 48 48"
      x="0px"
      xmlSpace="preserve"
      xmlns="http://www.w3.org/2000/svg"
      y="0px"
    >
      <polygon
        points="17.6,14.8 20.4,12 32.4,24 20.4,36 17.6,33.2 26.7,24 	"
      />
    </svg>
  </span>
</a>
`;

exports[`SectionLink should match snapshot without className 1`] = `
<a
  className="css-1uvd0mj "
  href="/"
>
  link
  <span
    className={undefined}
  >
    <svg
      version="1.1"
      viewBox="0 0 48 48"
      x="0px"
      xmlSpace="preserve"
      xmlns="http://www.w3.org/2000/svg"
      y="0px"
    >
      <polygon
        points="17.6,14.8 20.4,12 32.4,24 20.4,36 17.6,33.2 26.7,24 	"
      />
    </svg>
  </span>
</a>
`;
